### PR TITLE
Add Russian docs for 2.0

### PR DESCRIPTION
**Pull request by denji**

### CODE REVIEW

**Comment by bucaran** [comment]
@denji Thank you so much for this contribution.

@VEINHORN Coul you have a look? :)

File: docs/ru-RU/README.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+
+Установить из локального каталога.
+
+```sh
+fisher ~/my_aliases
+```
+
+Использовать в интерактивном-режиме. Редактировать fishfile и запустить `fisher`, чтобы удовлетворить изменения.
+
+> [Что такое fishfile и как я могу его использовать?](#6-what-is-a-fishfile-and-how-do-i-use-it)
```
Comment (bucaran) [inline docs/ru-RU/README.md]: Should link to this page :)

File: docs/ru-RU/README.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+
+```
+fisher help z
+```
+
+## Часто задаваемые вопросы
+
+### 1. Какая версия fish необходима?
+
+fisherman был построен для рыб >= 2.3.0. Если вы используете 2.2.0, добавьте следующий код в ваш `~/.config/fish/config.fish` для [сниппет](#8-what-is-a-plugin) поддержки.
```
Comment (bucaran) [inline docs/ru-RU/README.md]: сниппет should link to question 8 too :)

**Comment by demidovakatya** [comment]
Hey everyone! Thanks for a new portion of Russian docs! I've noticed a couple of weird things. For example: 

> fisherman был построен для рыб >= 2.3.0

Well... This means: "Fisherman was built for fish >= 2.3.0". For, like, _real_ fish. ( 🐟 )

Has anyone reviewed (or is reviewing) the texts? If not, please let me know – I'll do it.

**Comment by denji** [comment]
https://github.com/fisherman/fisherman/pull/227

**Comment by denji** [comment]
https://github.com/fisherman/fisherman/blob/master/docs/ru-RU/README.md

fisherman/fisherman#227

**Comment by bucaran** [comment]
Hi @demidovakatya, by all means do go ahead and send us a PR :)

**Comment by laughedelic** [comment]
Hi @demidovakatya. I just fixed that in #227. You're welcome to review it and improve further :wink:

That thing about literally translated "fish" was funny 😄👌

**Comment by demidovakatya** [comment]
Hi @laughedelic! I'll have a look, then. 👍

**Pull request merged**